shared/chat/conversation/list-area/start-conversation/index.js

```diff
@@ -5,15 +5,16 @@ import {styleSheetCreate, platformStyles, globalColors, globalMargins} from '../
 
 type Props = {
   participants: string,
+  showAddParticipants: boolean,
   onStart: () => void,
 }
 
 const StartConversation = (props: Props) => (
   <Box2 direction="vertical" style={styles.container} gap="small" gapStart={true} gapEnd={true}>
-    <Text type="BodySemibold" style={styles.header}>
-      You don't have an existing conversation with {props.participants} yet.
+    <Text type="BodySmall" style={styles.header}>
+      You haven't chatted with {props.participants} yet.
     </Text>
-    <Button type="Primary" label="Start a chat" onClick={props.onStart}>
+    <Button type="Primary" label="Start chatting" onClick={props.onStart}>
```

Comment:
If you're in a bad network, this appears to do nothing. It'd benefit from a waiting state. But also, since there's an input hint for `pendingWaiting`, maybe the view should be switching immediately to the conversation view and something isn't getting plumbed through?

Reply:
it can't switch as it can't allow you to type into it if its not made (the entire point of this pr). i removed pendingWaiting from input entirely

Reply:
i'll add a spinner to the button